jenkins/Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM jenkins/jenkins:2.129
+FROM jenkins/jenkins:latest
 ADD plugin/target/remoting-kafka.hpi /usr/share/jenkins/ref/plugins/remoting-kafka.jpi
 USER root
 RUN install-plugins.sh credentials
-RUN install-plugins.sh git
+RUN install-plugins.sh configuration-as-code
```

Comment:
Better to install plugins in a single line `RUN install-plugins.sh credentials configuration-as-code ...` or via using plugins.txt. I would also recommend to pin versions so that users of your demo do not get surprised by an incompatible plugin upgrade.

It is also a bit odd to see that `configuration-as-code` works without defining the experimental update center. It should not be available in the main update centers so far, only `alpha` versions have been released so far. But maybe the latest UC somehow picks them, http://updates.jenkins-ci.org/latest/ does not list it tho. CC @carlossg @ndeloof